src/libFLAC/cpu.c

```diff
@@ -245,9 +245,18 @@ ppc_cpu_info (FLAC__CPUInfo *info)
 #define PPC_FEATURE2_ARCH_2_07		0x80000000
 #endif
 
-	if (getauxval(AT_HWCAP2) & PPC_FEATURE2_ARCH_3_00) {
+	long hwcaps;
+#ifdef __linux__
+	hwcaps = getauxval(AT_HWCAP2);
+#elif defined (__FreeBSD__)
+	elf_aux_info(AT_HWCAP2, &hwcaps, sizeof(hwcaps));
+#else
+#error Unsupported platform! Please add support for reading ppc hwcaps.
+#endif
+	
+	if (hwcaps) & PPC_FEATURE2_ARCH_3_00) {
```

Comment:
It looks liek there's an unbalanced `)` here. Does this compile?

Reply:
You're right. Github makes it hard to paste code (since some time, it worked fine before) in its editing window so I just type code manually for short PR's...

Reply:
I see, yes. Normally travis would catch that sort of thing, but we don't test building for ppc.